### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Mythacri**
- Fixed Weapon property duplicates
- Commented out Steve language
- Moved information to language file

### CODE REVIEW

File: scripts/modules/system-config.mjs [hunk 1]
```diff
@@ -71,15 +71,14 @@ export class SystemConfig {
...
+      scatter: "MYTHACRI.weaponPropertyScatter"
+      superheavy: "MYTHACRI.weaponPropertySuperH"
 
       //Journeyman Properties
-      rocket: "Rocket"
-      heat: "Heat"
-      twinshot: "Twinshot"
+      rocket: "MYTHACRI.weaponPropertyRocket"
+      heat: "MYTHACRI.weaponHeat"
+      twinshot: "MYTHACRI.weaponTwinS"
```
Comment (krbz999) [inline scripts/modules/system-config.mjs:81]:
```diff
-      parrying: "MYTHACRI.weaponPropertyParry"
-      scatter: "MYTHACRI.weaponPropertyScatter"
-      superheavy: "MYTHACRI.weaponPropertySuperH"
-
-      //Journeyman Properties
-      rocket: "MYTHACRI.weaponPropertyRocket"
-      heat: "MYTHACRI.weaponHeat"
-      twinshot: "MYTHACRI.weaponTwinS"
+      parrying: "MYTHACRI.WeaponPropertyParry",
+      scatter: "MYTHACRI.WeaponPropertyScatter",
+      superheavy: "MYTHACRI.WeaponPropertySuperheavy",
+      //Journeyman Properties
+      rocket: "MYTHACRI.WeaponPropertyRocket",
+      heat: "MYTHACRI.WeaponPropertyHeat",
+      twinshot: "MYTHACRI.WeaponPropertyTwinshot"
```
Some minor style changes + missing commas would render this nonfunctional.

File: languages/en.json [hunk 2]
```diff
@@ -17,5 +17,11 @@
...
   "MYTHACRI.MayhemCurrentPoints": "You currently have {points} Mayhem Points.",
   "MYTHACRI.MayhemConsume": "Consume Mayhem Points",
-  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!"
+  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!",
+  "MYTHACRI.weaponPropertyParry": "Parrying",
+  "MYTHACRI.weaponPropertyScatter": "Scatter",
+  "MYTHACRI.weaponPropertySuperH": "Superheavy",
+  "MYTHACRI.weaponPropertyRocket": "Rocket",
+  "MYTHACRI.weaponHeat": "Heat",
+  "MYTHACRI.weaponTwinS": "Twinshot",
```
Comment (krbz999) [inline languages/en.json:26]:
```diff
-  "MYTHACRI.weaponPropertyParry": "Parrying",
-  "MYTHACRI.weaponPropertyScatter": "Scatter",
-  "MYTHACRI.weaponPropertySuperH": "Superheavy",
-  "MYTHACRI.weaponPropertyRocket": "Rocket",
-  "MYTHACRI.weaponHeat": "Heat",
-  "MYTHACRI.weaponTwinS": "Twinshot",
+  "MYTHACRI.WeaponPropertyParry": "Parrying",
+  "MYTHACRI.WeaponPropertyScatter": "Scatter",
+  "MYTHACRI.WeaponPropertySuperheavy": "Superheavy",
+  "MYTHACRI.WeaponPropertyRocket": "Rocket",
+  "MYTHACRI.WeaponPropertyHeat": "Heat",
+  "MYTHACRI.WeaponPropertyTwinshot": "Twinshot",
```

**Review by krbz999 — CHANGES_REQUESTED** [review]

File: scripts/modules/system-config.mjs [hunk 3]
```diff
@@ -71,14 +71,14 @@ export class SystemConfig {
...
+      parrying: "MYTHACRI.WeaponPropertyParry",
+      scatter: "MYTHACRI.WeaponPropertyScatter",
+      superheavy: "MYTHACRI.WeaponPropertySuperheavy",
 
       //Journeyman Properties
-      rocket: "MYTHACRI.weaponPropertyRocket"
-      heat: "MYTHACRI.weaponHeat"
-      twinshot: "MYTHACRI.weaponTwinS"
+      rocket: "MYTHACRI.WeaponPropertyRocket",
+      heat: "MYTHACRI.WeaponHeat",
```
Comment (krbz999) [inline scripts/modules/system-config.mjs]:
```diff
- heat: "MYTHACRI.WeaponHeat",
+ heat: "MYTHACRI.WeaponPropertyHeat",
```

**Review by krbz999 — CHANGES_REQUESTED** [review]

**Review by krbz999 — APPROVED** [review]
LGTM!

File: languages/en.json [hunk 4]
```diff
@@ -17,5 +17,11 @@
...
   "MYTHACRI.MayhemCurrentPoints": "You currently have {points} Mayhem Points.",
   "MYTHACRI.MayhemConsume": "Consume Mayhem Points",
-  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!"
+  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!",
+  "MYTHACRI.WeaponPropertyParry": "Parrying",
+  "MYTHACRI.WeaponPropertyScatter": "Scatter",
+  "MYTHACRI.WeaponPropertySuperheavy": "Superheavy",
+  "MYTHACRI.WeaponPropertyRocket": "Rocket",
+  "MYTHACRI.WeaponPropertyHeat": "Heat",
+  "MYTHACRI.WeaponPropertyTwinshot": "Twinshot",
```
Comment (krbz999) [inline languages/en.json:26]:
```diff
- "MYTHACRI.WeaponPropertyTwinshot": "Twinshot",
+ "MYTHACRI.WeaponPropertyTwinshot": "Twinshot"
```
A trailing comma in a json will cause the file to not load.

**Review by krbz999 — CHANGES_REQUESTED** [review]

File: languages/en.json (hunk 4 above)
Comment (Mythacri) [reply languages/en.json:26]: Okay NOW it should be fixed

**Pull request merged**